### PR TITLE
Add user requirement for CRDB-CLI

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/rs/references/crdb-cli-reference.md [hunk 1]
```diff
@@ -17,7 +17,14 @@ Each instance is responsible for updating the instances residing on other partic
 Write conflicts are resolved using [CRDTs]({{< relref "/rs/administering/active-active.md" >}}).
 
 To programmatically maintain an Active-Active database and its instances, you can use the CRDB-CLI command-line tool.
-With CRDB-CLI you can:
+
+## Using the crdb-cli
+
+To use the CRDB-CLI tool, you must log in to the terminal of an RS host with an account that is a member of the **redislabs** OS group.
```
Comment (AlonMagrafta) [inline content/rs/references/crdb-cli-reference.md:23]: With the customizable user and groups
maybe it's better to say you need to use a member who belongs to that group or if none defined then it's redisLabs

**Comment by bmansheim** [comment]
@banker Please review

File: content/rs/references/crdb-cli-reference.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ alwaysopen: false
 categories: ["RS"]
 ---
 An [Active-Active database]({{< relref "/rs/administering/active-active.md" >}}) (also known as CRDB or Conflict-free, Replicated DataBase)
-replicates your dataset across multiple, geographically distributed regions and allows read-write access in all locations.
-Active-Active databases are ideal for distributed applications that require the fastest response times, and also for disaster recovery.
+replicates your dataset across Redis Enterprise Software (RS) clusters that that are in geographically distributed regions.
+Active-Active databases allow read-write access in all locations so they are ideal for distributed applications that require the fastest response times, and also for disaster recovery.
```
Comment (banker) [inline content/rs/references/crdb-cli-reference.md:10]: Active-Active databases allow read-write access in all locations, making them ideal....

File: content/rs/references/crdb-cli-reference.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ alwaysopen: false
 categories: ["RS"]
 ---
 An [Active-Active database]({{< relref "/rs/administering/active-active.md" >}}) (also known as CRDB or Conflict-free, Replicated DataBase)
-replicates your dataset across multiple, geographically distributed regions and allows read-write access in all locations.
-Active-Active databases are ideal for distributed applications that require the fastest response times, and also for disaster recovery.
+replicates your dataset across Redis Enterprise Software (RS) clusters that that are in geographically distributed regions.
```
Comment (banker) [inline content/rs/references/crdb-cli-reference.md:9]: ...Redis Enterprise Software databases located in geographically distributed regions.

File: content/rs/references/crdb-cli-reference.md [hunk 4]
```diff
@@ -17,7 +17,14 @@ Each instance is responsible for updating the instances residing on other partic
...
 
 To programmatically maintain an Active-Active database and its instances, you can use the CRDB-CLI command-line tool.
-With CRDB-CLI you can:
+
+## Using the crdb-cli
+
+To use the CRDB-CLI tool, you must log in to the terminal of an RS host with a user that belongs to the group that RS was installed with (Default: **redislabs**).
+If you log in with a non-root user, you must add `/opt/redislabs/bin/` to your PATH environment variables.
+
+The syntax of CRDB-CLI is: `crdb-cli <command> <arguments>`.
```
Comment (banker) [inline content/rs/references/crdb-cli-reference.md:26]: CRDB-CLI commands use the following syntax: `crdb-cli <command> <arguments>`.

File: content/rs/references/crdb-cli-reference.md [hunk 5]
```diff
@@ -17,7 +17,14 @@ Each instance is responsible for updating the instances residing on other partic
 Write conflicts are resolved using [CRDTs]({{< relref "/rs/administering/active-active.md" >}}).
 
 To programmatically maintain an Active-Active database and its instances, you can use the CRDB-CLI command-line tool.
-With CRDB-CLI you can:
+
+## Using the crdb-cli
+
+To use the CRDB-CLI tool, you must log in to the terminal of an RS host with a user that belongs to the group that RS was installed with (Default: **redislabs**).
```
Comment (banker) [inline content/rs/references/crdb-cli-reference.md:23]: To use the CRDB-CLI tool, use SSH to log in to a RS host with a user belonging to the group that RS was installed with (Default: **redislabs**).

**Pull request merged**